src/mtev_http.c

```diff
@@ -1804,6 +1811,7 @@ _http_construct_leader(mtev_http_session_ctx *ctx) {
         }
       }
     }
+    if (!cl_present && keys != (const char **)static_key_array) free(keys);
```

Comment:
We don't want to free it yet - we use it later. From the code:

    if (!cl_present && keys != (const char **)static_key_array) free(keys);
  }
  qsort(keys, i, sizeof(*keys), casesort);
  kcnt = i;
  for(i=0;i<kcnt;i++) {
    int vlen;
    const char *key = keys[i], *value;

We use that keys value after the loop. We don't want to free it there if we're going to break out of the loop because we want to use it.